content/Arduino Cloud/Cloud Editor/If-you-see-an-error-when-performing-an-Over-the-Air-OTA-upload.md

```diff
@@ -48,12 +48,7 @@ In this section:
 
 This error indicates that the device's memory partition table is improperly formatted.
 
-* If you're using **Arduino GIGA R1 WiFi** or **Portenta H7**:
-  1. Reinitialize the storage with this Sketch: [QSPIFormat.ino](https://github.com/arduino/ArduinoCore-mbed/blob/main/libraries/STM32H747_System/examples/QSPIFormat/QSPIFormat.ino)
-  1. Reinstall the SSL certificates with this Sketch: [WiFiFirmwareUpdater](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/STM32H747_System/examples/WiFiFirmwareUpdater)
-
-* If you're using **Arduino Opta**: Follow [these instructions](https://docs.arduino.cc/tutorials/opta/memory-partitioning/#partitioning-the-memory-of-an-opta).
-* If you're using  **Portenta Machine Control**: Follow these [instructions](https://docs.arduino.cc/tutorials/opta/memory-partitioning/#partitioning-the-memory-of-a-portenta-machine-control).
+To resolve this issue, see [Reset the flash memory on STM32H747-based devices](https://support.arduino.cc/hc/en-us/articles/16206977438748-Reset-the-flash-memory-on-STM32H747-based-devices).
 
 > [!TIP]
 > If you're using GIGA R1 WiFi, see [Configure GIGA R1 WiFi for Over-The-Air (OTA) uploads](https://support.arduino.cc/hc/en-us/articles/12370721200540-Configure-GIGA-R1-WiFi-for-Over-The-Air-OTA-uploads).
```

Comment:
```suggestion
```
We can also remove this (serves the same purpose as [Reset the flash memory on STM32H747-based devices](https://support.arduino.cc/hc/en-us/articles/16206977438748-Reset-the-flash-memory-on-STM32H747-based-devices)) unless you see some specific point in keeping it here?

Reply:
[697e941](https://github.com/arduino/help-center-content/pull/457/commits/697e941807a332aaab63eb1e62678905a9ed4e59)